src/components/Newsfeed.vue

```diff
@@ -1,7 +1,7 @@
 <!-- eslint-disable vue/no-v-html -->
 <template>
 	<div class="newsFeed">
-		<div v-for="message in news" :key="message.key" class="message-box">
+		<div v-for="message in news" :key="message.subject" class="message-box">
 			<div class="message">
```

Comment:
This is causing a few warnings in the console because subjects from some messages are the same. We can start using the activity_id property as a key, since every event from the activity app has a different one and add this property to the hard coded data

Reply:
Ok, the news array does not have message objects with the property `activity_id` what do you mean?

Reply:
Since the final version will rely on the activities app, each message will have an unique `activity_id` property. We can add this property to the message objects just like the events from the `bookingNews` array on `utils.js` already have

Reply:
should be fixed in latest commit